app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CategoriesController < ApplicationController
+
+  def new
+    @category = Category.new
+  end
+
+  def create
+    @category = Category.new(category_params[:category])
+    @category.name.capitalize!
+    if @category.save
+      redirect_to user_products_path(current_user.id)
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def category_params
```

Comment:
A stronger way to write strong parameters would be 

`params.require(:category).permit(:name)`

This then requires that a category object be sent.
Here is the post that I used to verify my understanding before this comment:
http://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters